src/Microsoft.TestPlatform.CrossPlatEngine/Execution/MSTestV1TelemetryHelper.cs

```diff
@@ -79,14 +78,13 @@ internal static void AddTelemetry(TestResult testResult, IDictionary<string, int
 
                 if (adapterTelemetry.ContainsKey(key))
                 {
-                    adapterTelemetry[key] = adapterTelemetry[key] + 1;
+                    adapterTelemetry[key]++;
                 }
                 else
                 {
                     adapterTelemetry.Add(key, 1);
```

Comment:
```suggestion
                    adapterTelemetry[key] =  1;
```

Reply:
This is incorrect, it will try to lookup item with a key that does not exist. That is why there is the Add.

Reply:
You can use indexer syntax with new keys as well. Refer [docs](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.item?view=net-5.0)

Reply:
would you like me to change this or leave it as it is?

Reply:
If you can, so please do. This is not critical.